nodejs/week3/warmupExercise/index.js

```diff
@@ -0,0 +1,58 @@
+// Homework nodejs/week3 warmup section
+
+const express = require("express");
+const app = express();
+app.use(express.urlencoded({extended: true}));
+
+app.get("/", (req, res) => res.send("Welcome to Nodejs week3 warmup homework"));
+
+// Multiplication
+// http://localhost:3000/calculator/multiply?firstParam=2&secondParam=4&thirdParam=5
+// response: Multiplication of given numbers = 40
+app.get('/calculator/multiply', (request, response) => {
+    const queryParam = request.query;
+    let result = 1;
+    for (const parameter in queryParam){
+        if(isNaN(queryParam[parameter])){
+            return response.status(406).send('Unacceptable Input Parameter, Plz give numbers 😊');
+        }
+        result = result * Number(queryParam[parameter]);
+    }
+    response.send(`Multiplication of given numbers = ${result}`);
+})
+
+// Addition
+app.get('/calculator/add', (request, response) => {
+    const queryParam = request.query;
+    let result = 0;
+    for (const parameter in queryParam){
+        if(isNaN(queryParam[parameter])){
+            return response.status(406).send('Unacceptable Input Parameter, Plz give numbers 😊');
+        }
+        result = result + Number(queryParam[parameter]);
+    }
+    response.send(`The sum of given numbers = ${result}`);
+})
+
+// Getting data through the request body i.e request.body using POST (hint: use postman)
+app.post('/calculator/multiply', (request, response) => {
```

Comment:
I would expect to have a get method and a post method for each math operation.

Reply:
Thanks @NatsGt  for the review and the feedbacks, i will update them :)